File: app/src/main/java/com/dertyp7214/appstore/JSONParser.java
/*
 * Copyright (c) 2018.
 * Created by Josua Lengwenath
 */

package com.dertyp7214.appstore;

import org.json.JSONException;

public class JSONParser {

    private SimpleJSONObject jsonObject;

    public JSONParser(String json){
        try {
            this.jsonObject=new SimpleJSONObject(json);
        } catch (JSONException e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/com/dertyp7214/appstore/JSONParser.kt
/*
 * Copyright (c) 2018.
 * Created by Josua Lengwenath
 */

package com.dertyp7214.appstore

import org.json.JSONException

class JSONParser(json: String) {

    private var jsonObject: SimpleJSONObject? = null

    init {
        try {
            this.jsonObject = SimpleJSONObject(json)
        } catch (e: JSONException) {
            e.printStackTrace()
        }
    }
}
